apps/web/src/pages/Applications/ApplicationReviewPage/ReviewSection.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { Heading, Link } from "@gc-digital-talent/ui";
+import { useIntl } from "react-intl";
+
+interface ReviewSectionProps {
+  title: string;
+  path: string;
+  children: React.ReactNode;
+}
+
+const ReviewSection = ({
+  title,
+  path,
+  children,
+  ...rest
+}: ReviewSectionProps) => {
+  const intl = useIntl();
+  return (
+    <section data-h2-margin-bottom="base(x3)">
+      <div
+        data-h2-display="base(flex)"
+        data-h2-justify-content="base(space-between)"
+        data-h2-align-items="base(center)"
+        {...rest}
+      >
+        <Heading level="h4" data-h2-margin-top="base(0)">
+          {title}
+        </Heading>
```

Comment:
This should be an `h3`.

REF: https://webaim.org/techniques/semanticstructure/#headings

Reply:
Nice catch 👍 Fixed by [0c53549](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6600/commits/0c53549a5726b1da9a33d0c435318cd2d11214e5)